.vscode/settings.json

```diff
@@ -0,0 +1,18 @@
+{
+    "cSpell.words": [
+        "behaviour",
+        "Colour",
+        "customised",
+        "Deathmatch",
+        "gamemodes",
+        "gameplay",
+        "headshot",
+        "hitbox",
+        "loadout",
+        "respawn",
+        "respawning",
+        "respawns",
+        "unconnectable",
+        "Unported"
+    ]
+}
```

Comment:
I'm going to have to ask you to remove this file from the pull request as `.vscode` shouldn't be included in the repository. I will have to add a `.gitignore` entry for it.